ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -4,12 +4,33 @@
 //
 
 import UIKit
+import Combine
 
 // MARK: Tooltip
 
 /// A styled tooltip that is presented anchored to a view.
 @objc(MSFTooltip)
-open class Tooltip: NSObject {
+open class Tooltip: NSObject, TokenizedControlInternal {
+    var fluentTheme: FluentTheme {
```

Comment:
we still want the public vars on top and private var below grouped together? https://microsoft.github.io/swift-guide/FileOrganization.html